README.md

```diff
@@ -83,11 +83,29 @@ const account = Account.generate({ scheme: SingingSchemeInput.Ed25519, legacy: f
 #### Derive from private key
 
 ```ts
-const aptos = new Aptos();
-// This functions resolves the provided private key type and derives the public key from it
+// Create a private key instance based on the scheme
+const privateKey = new Ed25519PrivateKey("myprivatekeystring") | new Secp256k1PrivateKey("myprivatekeystring");
```

Comment:
This may be a little confusing, since this code would really compile.  Maybe... we comment out one, or we say one is privateKey1 and one is privateKey2.